### PR TITLE
Conflict resolved backmerge

### PR DESCRIPTION
**Pull request by turbomam**

### CODE REVIEW

**Comment by turbomam** [comment]
see also:
- https://github.com/microbiomedata/nmdc-schema/pull/1994
- https://github.com/microbiomedata/nmdc-server/issues/750

File: src/data/valid/Database-interleaved.yaml [hunk 1]
```diff
@@ -3932,13 +3945,13 @@ metagenome_sequencing_set:
...
-  - id: nmdc:wfmsa-99-qwertyuiop
+  - id: nmdc:wfmsa-99-qwertyuiop.1
     type: nmdc:MetagenomeSequencing
     name: Sequencing Activity for nmdc:mga0vx38
     has_input:
-      - nmdc:unvalidated_placeholder
+      - nmdc:procsm-99-abcdef
     has_output:
-      - nmdc:22afa3d49b73eaec2e9787a6b88fbdc3
+      - generic:xxx
```
Comment (aclum) [inline src/data/valid/Database-interleaved.yaml:3954]: @turbomam  generic:xxx should not be allowed. has_output should have a structured pattern for a data object valu ehere.

File: src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml [hunk 2]
```diff
@@ -97,25 +97,6 @@ biosample_set:
       term:
         type: nmdc:OntologyClass
         id: ENVO:00005792
-data_object_set:
```
Comment (aclum) [inline src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml:100]: Why did these lines get deleted?  This is one of the few/only examples where we have ref integrity w/in the example file so I'd like to keep intact.

File: src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml [hunk 3]
```diff
@@ -127,11 +108,9 @@ data_generation_set:
       - nmdc:dobj-12-yx0tfp52
     instrument_used:
       - nmdc:inst-14-xx07be40
-    part_of:
```
Comment (aclum) [inline src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml:130]: Keep this, it is an example of parthood for DataGeneration.

File: src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml (hunk 2 above)
Comment (aclum) [reply src/data/valid/Database-neon_Biosample_to_DataObject_NEON.yaml:100]: NM, i see the section just got moved.

File: src/data/valid/Database-nmdc-example.yaml [hunk 4]
```diff
@@ -67,68 +67,68 @@ study_set:
...
-    mod_date: 04-APR-20 08.26.35.067000000 AM
-    has_output:
-      - nmdc:dobj-00-mobydick
-    ncbi_project_name: Rhizosphere microbial communities from Carex aquatilis grown
-      in University of Washington, Seatle, WA, United States - 4-1-23 metaG
-    processing_institution: JGI
-    type: nmdc:NucleotideSequencing
-    associated_studies:
-      - nmdc:sty-00-avacado
+#data_generation_set:
```
Comment (aclum) [inline src/data/valid/Database-nmdc-example.yaml:70]: Why is all of this commented out?

File: src/data/valid/Database-nmdc-example.yaml [hunk 5]
```diff
@@ -284,19 +278,19 @@ biosample_set:
...
-    name: 11839.4.222578.GAGCTCA-TTGAGCT.fastq.gz
-    description: Raw sequencer read data
-    file_size_bytes: 34243309819
-    type: nmdc:DataObject
-  - id: nmdc:dobj-99-sQQw0I
-    name: 12660.4.274923.GATCGTAC-GTACGATC.fastq.gz
-    description: Raw sequencer read data
-    file_size_bytes: 7580035314
-    type: nmdc:DataObject
+#data_object_set:
```
Comment (aclum) [inline src/data/valid/Database-nmdc-example.yaml:281]: Why is this commented out?

File: src/data/valid/NucleotideSequencing-1.yaml [hunk 6]
```diff
@@ -13,8 +13,6 @@ has_output:
   - nmdc:dobj-00-9n9n9n
 ncbi_project_name: Thawing permafrost microbial communities from the Arctic, studying
   carbon transformations - Permafrost 712P3D
-part_of:
```
Comment (aclum) [inline src/data/valid/NucleotideSequencing-1.yaml:16]: We need to keep parthood examples for DataGeneration

File: src/schema/nmdc.yaml [hunk 7]
```diff
@@ -451,11 +489,20 @@ classes:
       id:
         required: true
         structured_pattern:
-          syntax: "{id_nmdc_prefix}:mixpro-{id_shoulder}-{id_blade}{id_version}{id_locus}"
+          syntax: "{id_nmdc_prefix}:mixpro-{id_shoulder}-{id_blade}"
```
Comment (aclum) [inline src/schema/nmdc.yaml:492]: missing closing $ on syntax

File: src/schema/workflow_execution_activity.yaml [hunk 8]
```diff
@@ -172,7 +172,11 @@ classes:
       id:
         required: true
         structured_pattern:
-          syntax: "{id_nmdc_prefix}:wfmsa-{id_shoulder}-{id_blade}{id_version}{id_locus}"
+          syntax: "{id_nmdc_prefix}:wfmsa-{id_shoulder}-{id_blade}{id_version}$"
+          interpolated: true
+      has_input:
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}:(bsm|procsm)-{id_shoulder}-{id_blade}$"
           interpolated: true
```
Comment (aclum) [inline src/schema/workflow_execution_activity.yaml:180]: we are missing an has_output pattern. This should be a DataObject or 'None' (to accommodate existing data in mongo)

File: src/data/invalid/Pooling-invalid_id-1.yaml [hunk 9]
```diff
@@ -0,0 +1,3 @@
+#invalid because ID has locus tag
+id: nmdc:poolp-11-547rwq94_scf_1000_c1_466_1380
+name: first pooling process
```
[the remark this answers is not in the document]
Comment (turbomam) [reply src/data/invalid/Pooling-invalid_id-1.yaml:3]: nice catch

**Pull request merged**